google/cloud/storage/benchmarks/BUILD

```diff
@@ -16,14 +16,60 @@ package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
 
-cc_binary(
-    name = "storage_latency_benchmark",
-    srcs = ["storage_latency_benchmark.cc"],
-    deps = ["//google/cloud/storage:storage_client"],
+load(":storage_benchmarks.bzl", "storage_benchmarks_hdrs", "storage_benchmarks_srcs")
+
+cc_library(
+    name = "storage_benchmarks",
+    srcs = storage_benchmarks_srcs,
+    hdrs = storage_benchmarks_hdrs,
+    deps = [
+        "//google/cloud:google_cloud_cpp_common",
+        "//google/cloud/storage:nlohmann_json",
+        "//google/cloud/storage:storage_client",
+        "@boringssl//:crypto",
+        "@boringssl//:ssl",
+        "@com_github_curl_curl//:curl",
+    ],
 )
 
-cc_binary(
-    name = "storage_throughput_benchmark",
-    srcs = ["storage_throughput_benchmark.cc"],
-    deps = ["//google/cloud/storage:storage_client"],
+load(
+    ":storage_benchmark_programs.bzl",
+    "storage_benchmark_programs",
```

Comment:
Remove trailing comma so it formats on one line like the others?

Reply:
Done.